nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/AWS_w1/NodeType.tosca

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypes" id="nodeTypesQuantil-AWS_w1">
```

Comment:
This should be part of https://github.com/OpenTOSCA/tosca-definitions-common

Reply:
Moved the AWS and Boto3 Node Types/Implementation [here](https://github.com/OpenTOSCA/tosca-definitions-common/pull/39).